### ISSUE TITLE
Duplicate module name: react-native-packager

### ISSUE THREAD

**Issue by samcorcos** [airbnb/lottie-react-native#65]
I've attempted to install Lottie via pod install, but when I run the app I get the following error in the packager:

```
Failed to build DependencyGraph: @providesModule naming collision:
  Duplicate module name: react-native-packager
  Paths: /Users/xxxx/MyApp/ios/Pods/React/packager/package.json collides with /Users/xxxx/MyApp/node_modules/react-native/packager/package.json

This error is caused by a @providesModule declaration with the same name across two different files.
Error: @providesModule naming collision:
  Duplicate module name: react-native-packager
  Paths: /Users/xxxx/MyApp/ios/Pods/React/packager/package.json collides with /Users/xxxx/MyApp/node_modules/react-native/packager/package.json

This error is caused by a @providesModule declaration with the same name across two different files.
    at HasteMap._updateHasteMap (/Users/xxxx/MyApp/node_modules/react-native/packager/react-packager/src/node-haste/DependencyGraph/HasteMap.js:158:13)
    at p.getName.then.name (/Users/xxxx/MyApp/node_modules/react-native/packager/react-packager/src/node-haste/DependencyGraph/HasteMap.js:133:31)
```

Any thoughts on how to resolve this naming collision?

**Comment by indivisable** [comment]
Could you try to install with npm?

> npm i --save github:airbnb/lottie-react-native
> react-native link lottie-ios
> react-native link lottie-react-native

**Comment by samcorcos** [comment]
@indivisable Thank you, that did solve the collision error, but now I get a different error. I cleared my `node_modules` and started from a clean project (which compiles fine without Lottie).

```
** BUILD FAILED **


The following build commands failed:

	Ld build/Build/Products/Debug-iphonesimulator/MyApp.app/MyApp normal x86_64
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/MyApp.app

An error was encountered processing the command (domain=MIInstallerErrorDomain, code=4):
Failed to chmod /Users/xxxx/Library/Developer/CoreSimulator/Devices/038CB145-979F-4AFB-A63E-5936AB7002DE/data/Library/Caches/com.apple.containermanagerd/Bundle/Application/A58C72C7-D1B7-4C74-A82D-8F54B7DBD342/MyApp.app/MyApp : No such file or directory
Launching org.reactjs.native.example.MyApp
```

That said, the app seems to open in the simulator and runs fine (I haven't tried using Lottie yet). Perhaps this error is not related to Lottie? Though, it only seems to appear after installing Lottie.

**Comment by lelandrichardson** [comment]
I believe this should now be fixed in `lottie-react-native@1.1.1`. Please try it out and let me know if you still have trouble. Closing this issue for now. Thanks!

**Comment by PimDeWitte** [comment]
I solved this by going in "project.pbxproj" and removing any references to modules I no longer use. In my case, it was libSplashScreen.a references that were interfering with libRCTSplashScreen.a resources from @remobile/react-native-splashscreen
